bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/BeneficiaryRow.java

```diff
@@ -183,8 +183,7 @@ public String toString() {
 		/**
 		 * Type: <code>CHAR</code>, max chars: 15. See <a href=
 		 * "https://www.ccwdata.org/cs/groups/public/documents/datadictionary/bene_id.txt">
-		 * CCW Data Dictionary: BENE_ID</a>, though note that this instance of +
-		 * * the field is encrypted.
+		 * CCW Data Dictionary: BENE_ID</a>.
```

Comment:
I think we still want to note here that this instance of the field is not encrypted.